arangod/RestHandler/RestVocbaseBaseHandler.h

```diff
@@ -219,6 +222,8 @@ class RestVocbaseBaseHandler : public RestBaseHandler {
 
   /// @brief the vocbase, managed by VocbaseContext
   TRI_vocbase_t& _vocbase;
+  
+  LogContext::Entry* _dbNameContext;
```

Comment:
As far as I can see, this variable is neither initialized in the header file nor in the constructor, so it will have an undefined value. We should initialize it to a nullptr!